content/_data/changelogs/weekly.yml

```
@@ -17990,6 +17990,174 @@
+        message: |-
+          Align the default value of the HTTP keep-alive timeout in Winstone with that of the upstream Jetty project by changing it from 5 seconds to 30 seconds.
+          Remove unused <code>--ajp13Port</code>, <code>--ajp13ListenAddress</code>, <code>--handlerCountMax</code>, and <code>--handlerCountMaxIdle</code> options.
+      - type: bug
+        category: bug
+        pull: 7155
+        issue: 69689
+        authors:
+          - ridemountainpig
+          - NotMyFault
+          - timja
+        pr_title:  "[JENKINS-69689] Table columns shadow is overlapping with other button shadow"
+        message: |-
+          Table column header highlight is overlapping with other header highlight.
+      - type: bug
+        category: bug
+        pull: 7140
+        issue: 69637
+        authors:
+          - loganmzz
+          - basil
+        pr_title: "[JENKINS-69637] Fix ClassCastException when param list is empty"
+        message: |-
+          Fix ClassCastException when trying to build a Job whose parameter list is empty.
+      - type: bug
+        category: bug
+        pull: 7270
+        authors:
+          - basil
+        pr_title: Unable to serialize `AtomicBoolean` on Java 17
+        references:
+        - url: https://github.com/jenkinsci/docker-plugin/issues/905
+          title: Unable to serialize AtomicBoolean on Java 17
+        message: |-
+          Allow serialization of atomic boolean data types on Java 17.
+      - type: rfe
+        category: developer
+        pull: 7275
+        authors:
+          - daniel-beck
+          - timja
+        pr_title: Update bundled script-security and workflow-support plugins
+        references:
+          - url: https://www.jenkins.io/security/advisory/2022-10-19/#SECURITY-2824%20(1)
+            title: 2022-10-19 Security Advisory
+          - url: https://www.jenkins.io/security/advisory/2022-10-19/#SECURITY-2881
+            title: 2022-10-19 Security Advisory
+        message: |-
+          Update bundled Script Security Plugin from 1172.v35f6a_0b_8207e to 1189.vb_a_b_7c8fd5fde.
+          Update bundled Pipeline: API Plugin from 1144.v61c3180fa_03f to 1164.v760c223ddb_32.
+          Update bundled Pipeline: Supporting APIs Plugin from 813.vb_d7c3d2984a_0 to 839.v35e2736cfd5c.
+          Update bundled SCM API Plugin from 602.v6a_81757a_31d2 to 608.vfa_f971c5a_a_e9.
+      - type: rfe
+        category: developer
+        pull: 7282
+        authors:
+          - dependabot[bot]
+        pr_title: Bump remoting from 3063.v26e24490f041 to 3068.v09b_895d8da_14
+        message: |-
+          Developer: Improve JNLP Port Unreachable error message with host value.
+      - type: rfe
+        category: developer
+        pull: 7274
+        authors:
+          - dependabot[bot]
+        pr_title: Upgrade Spring Security from 5.7.3 to 5.7.4
+        references:
+          - url:https://github.com/spring-projects/spring-security/releases/tag/5.7.4
```

```suggestion
          - url: https://github.com/spring-projects/spring-security/releases/tag/5.7.4
```
Let's make it valid yaml.